core/fsm/fsm.hpp

```diff
@@ -440,6 +442,12 @@ namespace fc::fsm {
                                source_state,          // source state
                                resulting_state.get());  // destination state
         }
+      } else {
+        // there were no rule for transition put event in queue in case it can
+        // be handled when initial state is changed
+        std::lock_guard lock(event_queue_mutex_);
+        event_queue_.push(event_pair);
+        tickAsync();
```

Comment:
Could it start endless loop in case there is only 'not ready to serve' events? Possible solutions - not to schedule tick or schedule delayed event.